### PR TITLE
Update the embedded postrgreSQL dependency

### PR DESCRIPTION
**Pull request by chtyim**
- The one from opentable is no longer actively maintained
- The io.zonky one is a fork and continuation of that
- Also upgraded to use  Postgres 11 such that tests can run locally in MacOS

### CODE REVIEW

File: cdap-app-fabric/pom.xml [hunk 1]
```diff
@@ -218,11 +218,10 @@
       <artifactId>jul-to-slf4j</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.opentable.components</groupId>
-      <artifactId>otj-pg-embedded</artifactId>
-      <scope>test</scope>
+      <groupId>io.zonky.test</groupId>
+      <artifactId>embedded-postgres</artifactId>
```
Comment (yaojiefeng) [inline cdap-app-fabric/pom.xml:222]: Should this still be in test scope?

**Review by yaojiefeng — APPROVED** [review]
Just one question, LGTM

File: cdap-app-fabric/pom.xml (hunk 1 above)
Comment (chtyim) [reply cdap-app-fabric/pom.xml:222]: The scope is set in the dependencyManagement section in the parent pom. That is to make sure all usages are in test scope unless explicitly overridden.

**Comment by tivv** [comment]
Wow, MacOS support, thank you so much @chtyim!

**Review by tivv — APPROVED** [review]

**Review by sechegaray — APPROVED** [review]

**Pull request merged**